archivers/libarchive/files/libarchive/archive_write_disk_posix.c

```diff
@@ -478,9 +478,11 @@ la_verify_filetype(mode_t mode, __LA_MODE_T filetype) {
 	case AE_IFLNK:
 		ret = (S_ISLNK(mode));
 		break;
+#ifdef S_ISSOCK
```

Comment:
Unless this goes upstream it's likely to get wiped in the next libarchive sync as it's really difficult to spot changes like this. Maybe put it in a patch instead?

Reply:
To my great surprise, it went upstream.